devices/pine64-pinephone-braveheart/kernel/default.nix

```diff
@@ -21,6 +22,6 @@
   installTargets = [ "install" "dtbs" ];
   postInstall = postInstall + ''
     mkdir -p "$out/dtbs/allwinner"
-    cp -v "$buildRoot/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtb" "$out/dtbs/allwinner/"
+    cp -v "$buildRoot/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone-${hardware.rev}.dtb" "$out/dtbs/allwinner/sun50i-a64-pinephone.dtb"
```

Comment:
This is where we should instead copy the versioned FDTs.

More things to look at:

 * https://gitlab.com/pine64-org/u-boot/-/commit/1f770b7060840137fd94af6fbf4788fcacc1a5fd
 * https://gitlab.com/pine64-org/u-boot/-/commit/b401e51cc1078ede81cc6f53d120f9c77a301952
 * https://gitlab.com/pine64-org/u-boot/-/commit/16d23194ec7eb3663bca1032eff1dc2b134cd4f8

These commits are proof that copying all Pinephone FDTs is the solution, and we should look into updating U-Boot soon :).